templates/statistics.html.twig

```diff
@@ -2,76 +2,83 @@
 
 {% block main %}
 
-<ol class="breadcrumb">
-	<li><a href="{{ path('home') }}">Home</a></li>
-	<li class="active">Statistics</li>
-</ol>
-<div class="row">
-	<aside class="col-md-3">
-		<h2>Recently Popular</h2>
-		<ol>
-			{% for book in recently_popular %}
-				<li value="{{ book.total }}">
-					<a href="{{ path('home', { lang: book.lang, page: book.title } ) }}" title="Download epub">
-						<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d5/EPUB_silk_icon.svg/20px-EPUB_silk_icon.svg.png" alt="The epub logo."/>
-					</a>
-					<a href="https://{{ book.lang }}.wikisource.org/wiki/{{ book.title }}" title="View on Wikisource">
-						{{ book.title|replace({'_': ' '}) }}
-					</a>
-				</li>
-			{% endfor %}
-		</ol>
-	</aside>
-	<div class="col-md-9">
-		<table class="table table-striped">
-			<caption>Stats for {{ month }}/{{ year }}</caption>
-			<thead>
-			<tr>
-				<th scope="col">Lang</th>
-				{% for format,value in total %}
-					<th scope="col">{{ format }}</th>
+<div class="container-fluid">
+	<ol class="breadcrumb">
+		<li><a href="{{ path('home') }}">{{ msg('breadcrumbs-home-link') }}</a></li>
+		<li class="active">{{ msg('breadcrumbs-stats-link') }}</li>
+	</ol>
+	<div class="row stats">
+		<div class="col-md-4 recents">
+			<h1 class=""><strong>{{ msg('recently-popular-heading') }}</strong></h1>
+			<h4 class="text-muted">{{ msg('recently-popular-subtext') }}</h4>
+			<ol>
+				{% for book in recently_popular %}
+					<li class="recent row">
+						<div class="col-md-9">
```

Comment:
This col and the one after it could perhaps be `col-xs-` ones so that the side-by-side alignment is kept on narrow screens. Although perhaps that's not great for long book titles. Anyway, can be done later! :)